course_dashboard/problem_stats/utils.py

```diff
@@ -89,8 +89,13 @@ def build_course_tree(module):
     if not module.get_children():
         if is_problem:
             is_open = True
+    # if a module has no display name. We need to get the default one through title().
+    if isinstance(module.display_name, (unicode, str)):
+        display_name = module.display_name
+    else:
+        display_name = module.display_name.title()
 
```

Comment:
je voulais dire que le comment pourrait être 
if display_name is not a string (pourquoi ?), it'a proxy (de quoi ?) we resolve by call title() method